manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "deletedvideos",
+  "version": "1.0",
+  "manifest_version": 2,
+  "content_scripts": [
+    {
+      "matches": ["*://*.youtube.com/playlist?list=*"],
```

```suggestion
      "matches": ["*://*.youtube.com/*"],
```
Should probably be changed to just "matches": ""*://*.youtube.com/*"] if we want to work playlists beyond the main playlist page. Check #3